feat(geo): update Geo to reflect React component release (#4190)
aws-amplify/docs: src/fragments/lib/geo/js/search.mdx

First, make sure you've provisioned a search index resource and configured your app using the instructions in either [Amplify CLI - Geo - Location Search](/cli/geo/search) or [Use existing Amazon Location Service resources](/lib/geo/existing-resources) and you have already setup [displaying a map](/lib/geo/maps) in your application.

<Callout>

> **Note:** If you are using React, you should use the [Amplify UI Geocoder component](https://ui.docs.amplify.aws/components/geo#geocoder) to display the search results on a map.

Reviewer: ```suggestion
**Note:** For React, you can use the [Amplify UI Geocoder component](https://ui.docs.amplify.aws/components/geo#geocoder) to display the search results on a map.
```
